### DISCUSSION TITLE
Icons for Insights or stats products

### DISCUSSION THREAD

**Discussion by seanmateer** [twilio-labs/paste#1255] [Request: Icon] [answered]
Hi 👋!

We have four icons we'd like to use for our upcoming [Deliverability Insights](https://twilio.invisionapp.com/share/5XZREP52NS3#/440813623_1-1_-_Overview) release. I'd propose adding these as "UI icons" so they can be used consistently across other insights products. 

The deliverability insights "chart-type" selection is based off of messaging insights (reference screenshot below), despite that messaging insights product is not 100% paste yet.
<img width="906" alt="Screen Shot 2021-03-04 at 4 59 58 PM" src="https://user-images.githubusercontent.com/4806849/110047425-1d065a80-7d0b-11eb-8bca-3e6f31f75a63.png"> 

I don't see a good way to link directly to streamline icons, so I'll include the exact names below and screenshots at the bottom. The icons we'd like to add are:
- **Pie Chart icon:** Streamline-regular "Analytics pie 1"
- **Bar Chart icon:** Streamline-regular "Analytics Bars"
- **Line Chart icon:** Streamline-regular "Analytics graph"
- **Tablular data icon:** Streamline-regular "Layout 2"

TBH I'm not _super_ psyched on the icons for line chart and tabular data compared to those in used in messaging insights and my current mockups, but cannot find better alternatives in streamline. Just curious if there is an opportunity to use custom icons in those cases if anyone agrees? Probably most important to align on something that can be used by all our products going forward though. 

Let me know if you think any of these are good candidates to be added to "UI icons" or if we should pursue the custom add approach. Thanks!

------------------------------

 **Reference icon images:**
<img width="277" alt="Screen Shot 2021-03-04 at 5 06 57 PM" src="https://user-images.githubusercontent.com/4806849/110047939-090f2880-7d0c-11eb-9b82-38c789bfd25e.png">
<img width="281" alt="Screen Shot 2021-03-04 at 5 11 44 PM" src="https://user-images.githubusercontent.com/4806849/110048242-b41fe200-7d0c-11eb-8f1d-dd992a68bab0.png">
<img width="277" alt="Screen Shot 2021-03-04 at 5 11 58 PM" src="https://user-images.githubusercontent.com/4806849/110048259-bbdf8680-7d0c-11eb-86b7-aa7d67746a42.png">
<img width="278" alt="Screen Shot 2021-03-04 at 5 12 13 PM" src="https://user-images.githubusercontent.com/4806849/110048275-c4d05800-7d0c-11eb-9caa-09825dbd37cf.png">

**Comment by serifluous** [comment]
Nice @seanmateer , thanks for including the Streamline icons! Here's what I'm thinking for icon names:

- DataPieChartIcon
- DataBarChartIcon
- DataLineChartIcon
- DataTableIcon

We'll see what we can grab or edit from Streamline, and we'll tag you for review in Figma. I'm personally not super jazzed about the Streamline pie chart icon either compared to the legacy one.

How soon does your engineering team need these? If it's within the next 3 weeks, we'll send the SVGs to you, and your team can go the [self-service route](https://paste.twilio.design/icons/how-to-add-an-icon#adding-the-icon-to-the-paste-repository). Otherwise, we'll likely plan it into our next 2 sprints (next sprint starts Wed, March 10).

**Reply by seanmateer** [reply]
Awesome! Thank you for the super quick reply @serifluous. There are some other pie variations on streamline, but I thought that one was least offensive. The others I saw are more embellished, but please feel free to suggest an alternative! 

Eng is prioritizing some UI fit and finish in their current and next sprints before onboarding our first pilot customer, which we are targeting for the first week of April. If it's not too much trouble, would it be possible to get the SVGs in the short-term just in case we need them and then update to paste icons once available?

**Reply by seanmateer** [reply]
@serifluous Just following up as eng is eager to get their hands on these - they are okay with taking the self-service route for now. Any chance you can share the SVGs our way in the short term? cc @GiantRobots

**Reply by serifluous** [reply]
(followed up in slack)

**Comment by serifluous** [comment, marked as answer]
Icons have been added https://paste.twilio.design/icons
